spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/db/migration/mysql/V1__Initial_Setup.java

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
```

Comment:
Nitpick but it would be 2022-2022.    
I had to pick on something ;-)

Reply:
Yep - fair enough. All of these files were actually here since 2019 and recently deleted like 6 months ago. I just resurrected these from git so was unsure if we should leave the start dates like they were? Thoughts?